packages/firebase/README.md

```
@@ -264,37 +280,48 @@ The new API provides:
 ### Common Issues
 
 1. **Authentication Errors**
+
    ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)